src/edu/csus/ecs/pc2/core/list/RunList.java

```diff
@@ -69,15 +65,15 @@ private int getINIBaseRunNumber() {
                 if (baseNum == null || baseNum.equals("")) {
                     retVal = 1; 
                 } else {
-                    //SOMEDAY: remove the following line and log a message instead
+                    //SOMEDAY: remove the following line and log a message instead (note that the problem is lack of access to the controller/log)
                     System.out.println ("Found server.baseRunNumber in .ini file: '" + baseNum + "'");
                     retVal = Integer.parseInt(baseNum);
                     if (retVal < 1) {
                         throw new NumberFormatException("Negative base not allowed: " + baseNum);
                     }
                 }
             } catch (NumberFormatException e) {
-                System.err.println ("Illegal base run number value in INI file: '" + IniFile.getValue("server.baseRunNumber") + "'");
+                System.err.println ("Illegal base run number value in INI file: '" + IniFile.getValue("server.baseRunNumber") + "'; defaulting to base run number = 1");
```

Comment:
I don't disagree that this should be handled with logging instead of println's.  However, all I did  was add an additional explanatory comment to help the user understand the output ("continuous improvement").  Adding logging would not be part of the bug this PR was intended to fix and I didn't want to spend the additional time to deal with logging (there's a limit to how much "continuous improvement" I'm willing to tackle when I'm in the middle of trying to get a bug fixed and approved...)